scripts/config/msu/classes/timer.nut

```diff
@@ -2,22 +2,49 @@
 {
 	ID = null;
 	Start = null;
+	PauseStart = null;
+	PauseIncrement = null;
 
 	constructor(_id)
 	{
 		this.ID = _id;
 		this.Start = ::Time.getExactTime();
+		this.PauseIncrement = 0;
 	}
 
 	function get(_msg = "", _stop = false)
 	{
-	    local time = (::Time.getExactTime() - this.Start) * 1000;
+		// Make sure to substract paused timme
+		if (this.PauseStart != null)
+			this.unpause();
```

Comment:
This is a bit of a question, since technically you can get() the current time at any point, and now this would unpause the timer. Instead it could just substract the incremented time or something. Suggestions?

Reply:
use a local variable here for pauseIncrement that you set differently based on whether the timer is currently paused